manifests/g/Google/WebDesigner/1.3.36.122/Google.WebDesigner.locale.en-US.yaml

```diff
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Google.WebDesigner
+PackageVersion: 1.3.36.122
+PackageLocale: en-US
+Publisher: Google LLC
+PublisherUrl: https://withgoogle.com
+PublisherSupportUrl: https://support.google.com/webdesigner#topic=3249465
+PrivacyUrl: https://policies.google.com/privacy?hl=
+Author: Google
+PackageName: Google Web Designer
+PackageUrl: https://webdesigner.withgoogle.com/
+License: Copyright 2018 Google LLC
+LicenseUrl: https://policies.google.com/terms?hl=
+Copyright: https://policies.google.com/terms?hl=
+ShortDescription: Create engaging, interactive HTML5-based designs and motion graphics that can run on any device.
+Description: Google Web Designer gives you the power to create beautiful, engaging HTML5 content. Use animation and interactive elements to bring your creative vision to life, and enjoy seamless integration with other Google products, like Google Drive, Display & Video 360, and Google Ads.
+Moniker: Google Web Designer
+Tags:
+- google
+- web
+- web designer
+- website maker
+- web bulider
+- free
+Agreements:
+- AgreementUrl: https://policies.google.com/terms
+- AgreementUrl: https://policies.google.com/privacy

```

Comment:
This can only be used by Google employees.

Reply:
Are you sure? I can download it and use it fine from a public landing page (https://webdesigner.withgoogle.com)

There's a forum and everything: https://support.google.com/webdesigner/threads?hl=en&thread_filter=(category:getting_started)

Reply:
The `Agreements` field isn't meant contributors; only publishers.

Reply:
Oh, I thought you meant the app in general, not the field. Sorry!